css/layout.css

```diff
@@ -0,0 +1,145 @@
+#a-box {
+  width: 480px;
+  height: 75px;
+  background-color: rgb(20, 108, 184);
+}
+
+#b-box {
+  width: 480px;
+  height: 150px;
+  background-color: aqua;
+}
+
+#c-box {
+  width: 480px;
+  height: 125px;
+  background-color: aquamarine;
+}
+
+#d-box {
+  width: 480px;
+  height: 125px;
+  background-color: rgb(43, 152, 74);
+}
+
+#e-box {
+  width: 240px;
+  height: 250px;
+  background-color: blue;
+}
+
+ #f-box {
+  width: 240px;
+  height: 250px;
+  background-color: blueviolet;
+}
+
+#g-box {
+  width: 240px;
+  height: 250px;
+  background-color: blanchedalmond;
+}
+
+#h-box {
+  width: 240px;
+  height: 250px;
+  background-color: black;
+}
+
+#i-box {
+  width: 480px;
+  height: 80px;
+  background-color: brown;
+}
+
+#cd-container{
+  width: 480px;
+  height: 250px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+#efgh-container {
+  width: 480px;
+  height: 500px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+@media(min-width: 768px){
+  * {

```

Comment:
since your `*` selector is in base.css, i would have the media query for this selector also in that file.